package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "stacked-sheet",
+  "main": "src/stacked-sheet.js",
+  "module": "src/stacked-sheet.js",
+  "files": [
+    "src"
+  ],
+  "version": "0.0.0",
+  "description": "Webcomponent stacked-sheet with lit-element and open-wc recommendations",
+  "license": "MIT",
+  "dependencies": {
+    "lit-element": "^2.0.1"
+  },
+  "lint-staged": {
```

Comment:
Where or how do we use `lint-staged`?

Reply:
Good spot. We don't use it, it's a leftover from the pwa app